convert-to-declarative-plugin/src/main/java/io/jenkins/plugins/todeclarative/converter/jobproperty/ParameterPropertyConverter.java

```diff
@@ -88,7 +89,10 @@ protected ModelASTBuildParameter build( ParameterDefinition parameterDefinition,
         return null;
     }
 
-
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification = "Superclass method is @CheckForNull, but subclass impl never returns null")
+    private boolean getDefaultBooleanParameterValue(BooleanParameterDefinition def) {
```

Comment:
Awkward, but seems worth it to be able to enable SpotBugs. I guess we could instead add an if/else with a default value or an exception or something, but that seems awkward too.